packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -129,6 +135,7 @@ function WorklistDataGrid(props) {
     const elementWithTabStop = grid.current.rows[nextRow].cells[nextCol];
     elementWithTabStop.tabIndex = 0;
```

Comment:
Should this now be handled during rendering since you have a useState hook now?  Would the focus be set in the useEffect hook?

Reply:
I've removed the cell select functionality so this is resolved